### PR TITLE
feature/Mjlighet att automatiskt visa quicklinks efter frsta blocket p sidan CU 865c4ruqr

### PR DESCRIPTION
**Pull request by annalinneajohansson**
- String translations
- Changing the quicklinks placement ACF field to account for more placement options
- Refactored displayQuicklinksAfterContent()
- Cleaned up methods relating to placement of quicklinks 
- Renamed variables
- Handle displaying quicklinks menu after first block on page when option is selected

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-865c4ruqr Möjlighet att automatiskt visa quicklinks efter första blocket på sidan](https://app.clickup.com/t/865c4ruqr)

File: assets/source/3.0/sass/navigation/_quick-links.scss [hunk 1]
```diff
@@ -30,8 +29,9 @@ $c-quicklinks-gap: var(--c-quicklinks-gap, 2) !default;
 
     &.c-header--sticky {
         top: -1px;
-        position: relative;
-        &.is-stuck {
+        <<<<<<< HEAD=======position: relative;
+
```
Comment (sebastianthulin) [inline assets/source/3.0/sass/navigation/_quick-links.scss:33]: Merge conflict.

**Comment by sebastianthulin** [comment]
Approved. But fix merge conflict in styles.

File: assets/source/3.0/sass/navigation/_quick-links.scss (hunk 1 above)
Comment (annalinneajohansson) [reply assets/source/3.0/sass/navigation/_quick-links.scss:33]: [Of course there is.](https://media.giphy.com/media/wID3zXURLH1jrjCcZy/giphy.gif)

**Pull request merged**